ddb_cp.module

```diff
@@ -243,19 +255,21 @@ function ddb_cp_ajax_test_result($site = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @todo: Missing return statement it will note return any thing if the if statement is hit?
```

Comment:
I see you point here. But drupal_json_output() don't finishes the http request it simply sets a header and echo the json to standard out. So if one calls you function $json = ddb_cp_ajax_test_result() it don't return anything, so the documentation is not correct.